modules/Microsoft.VirtualMachineImages/imageTemplates/.test/common/deploy.test.bicep

```
@@ -89,5 +89,6 @@ module testDeployment '../../deploy.bicep' = {
     unManagedImageName: '<<namePrefix>>-umi-${serviceShort}-001'
     userMsiResourceGroup: resourceGroupName
     vmSize: 'Standard_D2s_v3'
+    stagingResourceGroup: ''
```

I guess - just for fun - we could provide a value?
```suggestion
    stagingResourceGroup: '<<namePrefix>>${serviceShort}001-rg'
```